doc/data/messages/b/bad-mcs-method-argument/details.rst

```diff
@@ -1 +1,7 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
+Used when a metaclass method has a first argument named differently than the
+value specified in valid-classmethod-first-arg option (default to `cls`),
+recommended to easily differentiate them from regular instance methods.
+
+Weird enough but usually naming of the first argument is `mcs` and pylint
+documentation says so but when someone specifies it then the lib throws
+C0203 error.
```

Comment:
There's a description in the message directly in the checker, you should probably modify that and regenerate the doc :)

Reply:
Changed documentation to what I found in "doc/user_guide/checkers/features.rst"